tests/drivers/i2s/i2s_api/src/common.c

```diff
@@ -111,11 +111,10 @@ static int tx_block_write_slab(const struct device *dev_i2s, int att, int err,
 	char tx_block[BLOCK_SIZE];
 	int ret;
 
-	fill_buf((uint16_t *)tx_block, att);
+	fill_buf((uint32_t *)tx_block, att);
 	ret = i2s_buf_write(dev_i2s, tx_block, BLOCK_SIZE);
 	if (ret != err) {
-		TC_PRINT("Error: i2s_write failed expected %d, actual %d\n",
-			 err, ret);
+		TC_PRINT("Error: i2s_write failed expected %d, actual %d\n", err, ret);
```

Comment:
Such changes in formatting should not be in the same commit as changes modifying the behavior of the test. Are they needed at all?

Reply:
Does this make those changes needed?

Reply:
To be removed